### DISCUSSION TITLE
アイテム追加案

### DISCUSSION THREAD

**Discussion by deskull-m** [sikabane-works/bakabakaband#197] [Ideas]
思いついたもんと適当に通ずるっこんでいく。どこまでネタとして許されるかは都度考えさせていただきます。

**Comment by deskull-m** [comment]
とりあえず今考えたけどまだ実装されてないもの
 * アイスティーの薬
 * イチジク浣腸の薬
 * ★邪剣『夜』
 * ★聖剣『月』

**Comment by RL-Nadir** [comment]
汚いエゴを提案します。これ、なにげにガチャピン対策になりそうですね。

N:114514:拒むことを知らない
E:of *DRAGON*
X:31:13
W:0:13:0:7500
C:0:0:11:3
F:CON | CHR |
F:NASTY |
F:RES_POISON | RES_CHAOS | ESP_NASTY | AGRAVATE_NASTY |
F:EXTRA_RES |

確率で、くっそ汚いのを呼び寄せる呪いがつくとなお良いと思います。

AGRAVATE_NASTY　はNASTY限定反感です。
現状このようなものはありませんが、悪魔が絶対に敵対する神聖装備みたいな拡張ができるかもしれません。

**Comment by RL-Nadir** [comment]
汚い武器エゴもついでに。

N:810:ものすごい太い
E:of Slay Nasty
X:24:19
W:0:19:0:15000
C:-8:19:0:3
F:STR | CHR | NASTY
F:SLAY_NASTY |
F:RES_POISON | EXTRA_RES
F:ESP_NASTY | AGGRAVATE_NASTY
F:XTRA_DICE | XTRA_POWER

確率で混沌ブランドか毒殺（ポジ）が付いてもいいかもしれません。

**Reply by deskull-m** [reply]
ああ＾～どっちもいいっすねえ～。とりあえずエゴ追加にもてェつけてみますわ。
